src/typescript/frontend/src/components/trade/DepositWithdrawModal/SelectMarketContent.tsx

```diff
@@ -1,116 +1,490 @@
+                      ) {
+                        header.column.setFilterValue(undefined);
+                      }
+
+                      // setting filter
+                      if (
+                        header.id === "name" &&
+                        // need these checks to prevent infinite loop
+                        // table sets empty string as undefined so we need to check for both
+                        header.column.getFilterValue() != filter &&
+                        filter != ""
+                      ) {
+                        header.column.setFilterValue(filter);
+                      }
+                      if (header.id === "name") {
+                        if (
+                          filter == "" &&
+                          header.column.getFilterValue() != undefined
+                        ) {
+                          header.column.setFilterValue(undefined);
+                        }
+                        if (
+                          filter != "" &&
+                          header.column.getFilterValue() != filter
+                        ) {
+                          header.column.setFilterValue(filter);
+                        }
+                      }
+
+                      // recognized
+                      if (header.id === "recognized") {
+                        if (
+                          selectedTab === 0 &&
+                          header.column.getFilterValue() == undefined
+                        ) {
+                          header.column.setFilterValue(true);
+                        }
+                        if (
+                          selectedTab === 1 &&
+                          header.column.getFilterValue() == true
+                        ) {
+                          header.column.setFilterValue(undefined);
+                        }
+                      }
+                      return (
+                        <th
+                          className={`${i === 0 ? "text-left" : ""} ${
+                            header.id === "recognized" ||
+                            (header.id === "24h_change" && "text-center")
+                          }
+          ${i === 0 ? TABLE_SPACING.paddingLeft : ""}
+          ${
+            i === headerGroup.headers.length - 1
+              ? TABLE_SPACING.paddingRight
+              : ""
+          } `}
+                          key={header.id}
+                        >
+                          {header.isPlaceholder
+                            ? null
+                            : flexRender(
+                                header.column.columnDef.header,
+                                header.getContext()
+                              )}
+                        </th>
+                      );
+                    })}
+                  </tr>
                 ))}
-              </tr>
-            ))
-          )}
-        </tbody>
-      </table>
+              </thead>
+              <tbody>
```

Comment:
Modal should be fixed height and have the table body overflow on the y-axis when there are too many markets to fit on screen. Currently, the entire modal just scrolls, which isn't ideal

Reply:
yeah expected behavior is to have the table scroll imo

Reply:
will likely just set a minheight, but need to account for mobile screensize if height is < modal height